### ISSUE TITLE
Update not available

### ISSUE THREAD

**Issue by FlashWall** [Triky313/AlbionOnline-StatisticsAnalysis#270]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Tool Version [e.g. 5.0.0]
 - NPcap Version [e.g. 1.6]

**Additional context**

**Comment by FlashWall** [comment]
My loot loger was doing great until an update came out, I did it then it started to ping me .4 items, swiftclaws as morgana nightmare ect.
I tried to uninstall everything then re install it, but still not working.
I tried to update it but it's not working as well, it doesnt even open something when i click to update it.
Help me !
Thanks

**Comment by Triky313** [comment]
I need more information! You haven't filled in anything that might be useful.
Is there an error log in the tools folder, if so send it to me.

Which Windows are you using?
On which server do you play?

**Comment by JinYanyaaa** [comment]
I have selected the update prompt to be prompted after 7 days, but now I want to update in advance and cannot find the update program

**Comment by Triky313** [comment]
You can find a big update button in settings.

**Comment by FlashWall** [comment]
> I need more information! You haven't filled in anything that might be useful. Is there an error log in the tools folder, if so send it to me.
> 
> Which Windows are you using? On which server do you play?

Ok, so i am with windows 10 and playing on the west server.
There is no error log.
I just made an update of the app few days ago and right after that the problems started.
I use it for zvz loots.
Now today the thing is that it's only what i loot in the logs and not my guild mates.
I've tried to uninstall the app and re download it but it's not working.
When I go to the settings and press the button to update the app nothing happens. 
What can i do to have a functional version of the app.
Thanks
